nhais/outbound/schema/validate_request.py

```diff
@@ -0,0 +1,25 @@
+from fhir.resources.fhirabstractbase import FHIRValidationError
+from jsonschema import ValidationError
+
+from fhir.resources.fhirelementfactory import FHIRElementFactory
+
+import utilities.integration_adaptors_logger as log
+from outbound.schema.schema_validation_exception import SchemaValidationException
+
+_logger = log.IntegrationAdaptorsLogger(__name__)
+
+
+def validate_patient(request_body):
+    try:
+        patient = FHIRElementFactory.instantiate('Patient', request_body)
+        __validate_patient_id_exists_in_payload(request_body)
+    except ValidationError as e:
```

Comment:
Does FHIRElementFactory raise ValidationError? I suspect this is old JSONSchema code

Reply:
Removed